internal/document_test.go

```diff
@@ -1,14 +1,53 @@
 package internal_test
 
 import (
+	"crypto/x509"
+	"encoding/base64"
 	"testing"
 	"time"
 
-	"github.com/stretchr/testify/assert"
-
 	"github.com/blocky/nitrite/internal"
+	"github.com/blocky/nitrite/mocks"
+	"github.com/fxamacker/cbor/v2"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
+func initDocuments(t *testing.T) (
```

Comment:
Ok I put the majority of the logic into a loop but I'm still explicitly returning three documents because the tests assume a specific number and order for the returned documents (i.e., nitro, debug, then self-signed)